src/commands/setup/prep-charts.ts

```diff
@@ -677,6 +677,38 @@ export default class SetupPrepCharts extends Command {
         }
       }
 
+      else if (chartName == 'l1-interface') {
+        if (!productionYaml.configMaps?.env?.data) {
+          this.error(`${chartName}: configMaps.env.data not found in config`);
+        }
+
+        const todoMappings = {
+          "DOGEOS_L1_INTERFACE_NETWORK_STR": this.withdrawalProcessorConfig["network_str"],
+          "DOGEOS_L1_INTERFACE_SCROLL_MESSENGER_ADDRESS": this.getConfigValue("contractsFile.L1_SCROLL_MESSENGER_PROXY_ADDR"),
+          "DOGEOS_L1_INTERFACE_L1_GENESIS_BLOCK": this.dogeConfig.defaults?.dogecoinIndexerStartHeight,
+          "DOGEOS_L1_INTERFACE_L2_MOAT_CONTRACT_ADDRESS": this.getConfigValue("contractsFile.L2_MOAT_IMPLEMENTATION_ADDR"),
```

Comment:
This service should only need to know of the proxy address -- Not sure which ENVVAR gets used in the service itself though, but it should be set to the proxy value.